development/image-url-helper/README.md

```diff
@@ -4,65 +4,56 @@
 
 Image URL Helper is a tool that provides the following subcommands:
 
-* The `check` command finds all Helm chart images that don't use the `imageurl` template.
-* The `list` command lists all Helm chart images by checking the `values.yaml` files.
-* The `missing` command lists all non-existent Helm chart images by checking the `values.yaml` files.
-* The `promote` command updates the container registry path and Helm chart images versions in the `values.yaml` files. The subcommand also prints a YAML that can be used by the [Image Syncer](../image-syncer) tool to promote images.
-* The `components` command generates a [Component-Descriptor](https://github.com/gardener/component-spec) file for the `kyma` repository from `values.yaml` files.
-
+- The `check` command finds all Helm chart images that don't use the `imageurl` template.
+- The `list` command lists all Helm chart images by checking the `values.yaml` files.
+- The `missing` command lists all non-existent Helm chart images by checking the `values.yaml` files.
+- The `promote` command updates the container registry path and Helm chart images versions in the `values.yaml` files. The subcommand also prints a YAML that can be used by the [Image Syncer](../image-syncer) tool to promote images.
```

Comment:
The link you use takes you to the doc where image-syncer is not capitalized. For the sake of consistency, I'd use "image-syncer" rather than "Image Syncer".